pkg/timeline/service/push.ts

```diff
@@ -3,20 +3,38 @@ import { Result } from '@mikuroxina/mini-fn';
 import type { AccountModuleFacade } from '../../intermodule/account.js';
 import type { Note } from '../../notes/model/note.js';
 import type { TimelineNotesCacheRepository } from '../model/repository.js';
+import type { FetchSubscribedListService } from './fetchSubscribed.js';
 import type { NoteVisibilityService } from './noteVisibility.js';
 
 export class PushTimelineService {
   constructor(
     private readonly accountModule: AccountModuleFacade,
     private readonly noteVisibility: NoteVisibilityService,
     private readonly timelineNotesCacheRepository: TimelineNotesCacheRepository,
+    private readonly fetchSubscribedListService: FetchSubscribedListService,
   ) {}
 
   /**
    * @description Push note to home timeline
    * @param note to be pushed
    * */
   async handle(note: Note): Promise<Result.Result<Error, void>> {
+    const home = await this.pushToHomeTimeline(note);
+    if (Result.isErr(home)) {
+      return home;
+    }
+
+    const list = await this.pushToList(note);
+    if (Result.isErr(list)) {
+      return list;
+    }
+
+    return Result.ok(undefined);
```

Comment:
```suggestion
    return this.pushToList(note);
```
この実行結果をそのまま `return` してもよさそうです. ただし `pushToList()` の戻り値型が明示されておらず読みづらかったので, 明記して起きたほうがよさそうです.